--- gnome-online-accounts-3.46.0/meson.build~	2022-09-17 19:09:28.000000000 +0000
+++ gnome-online-accounts-3.46.0/meson.build	2022-10-01 17:59:30.509735227 +0000
@@ -131,7 +131,7 @@
 
 enable_fedora = get_option('fedora')
 if enable_goabackend and enable_fedora
-  gcr_dep = dependency('gcr-3')
+  gcr_dep = dependency('gcr-4')
   krb5_dep = dependency('krb5')
 
   config_h.set('GCR_API_SUBJECT_TO_CHANGE', true)
